changes/2052.doc.md

```diff
@@ -0,0 +1 @@
+docs: add the manager's RPC client certificate entry on "install from package" document
```

Comment:
I think change logs is not a commit message so prefixes like `docs:` are not required.